netsuite/11.3/modules/ROOT/pages/netsuite-examples.adoc

```diff
@@ -542,22 +542,46 @@ payload.ns0#record
 
 == Custom Records and Fields
 
-The following examples show how to add, get, and delete custom records with
-custom fields.
+The following example shows how to add, get, and delete custom records with
+custom fields using DataSense.
+
+The following screenshot shows the Studio app flow for this example:
+
+image::custom-record-flow.png[App flow for the Custom Records example]
+
+Creating this example involves adding the *Add* operation, *Set Variable*
+component, *Get* operation, and *Delete* operation.
 
 [[add-custom-record]]
 === Add a Custom Record
 
-This example shows how to add a custom record with custom fields using the *Add*
-operation. This example is generated using DataSense.
+Add a custom record with custom fields using the *Add*
+operation.
+
+Include the following attributes to add a custom record:
+
+* *separators*
++
+Specifies a separator character.
+
+* *recordType*
++
+References the custom record type.
+
+* *scriptId*
++
+References the script ID of the record.
+
+* *internalId*
++
+References the internal ID of the record.
+
+* *XSI type*
++
+Specify the XSI type as *CustomRecordRef* to add a custom record.
 
-Pick a type in the operation to generate metadata. In this example, the
-key `\\__customRecordType__customrecord_2663_bank_details\__374` is used. Keys consist of
-*separators*, *recordType*, *scriptId*, and *internalId*. For this example, the
-separator is `__`, the recordType is `customRecordType`, the scriptId is
-`customrecord_2663_bank_details`, and the internalId is `374`.
```

Comment:
I think we should keep this example where is shown the key name structure. Or add the example values to each corresponding attribute from above.

Reply:
done